website/content/docs/architecture/catalog/v2.mdx

```diff
@@ -22,38 +22,68 @@ When Consul registers services, it records [user-defined and Consul-assigned inf
 - Locations of the _nodes_ the instances run on
 - Names of the _services_ the instances are associated with
 
-This information enables Consul to associate service names with the individual instances and their unique network addresses, and it is essential to Consul’s service discovery and service mesh operations.
+This information enables Consul to associate service names with the individual instances and their unique network addresses, which makes it essential to Consul’s service discovery and service mesh operations.
 
-The [Consul v1 catalog API](/consul/docs/architecture/catalog/v1) was designed prior to the introduction of Consul’s service mesh features. Communication in Consul’s service mesh is secured through Consul's ACL system, which requires that a Kubernetes ServiceAccount resource match the Service name. As a result, only one service can represent a service instance in the v1 catalog.
+The [Consul v1 catalog API](/consul/docs/architecture/catalog/v1) was designed prior to the introduction of Consul’s service mesh features. One major implication of this design is that communication in Consul’s service mesh is secured through Consul's ACL system, which requires that a Kubernetes ServiceAccount resource match the Service name. As a result, only one service can represent a service instance in the v1 catalog.
```

Comment:
To make it clear, shall the last sentence be "As a result, one Kubernetes service can represent a service instance in the v1 catalog."?